### ISSUE TITLE
[Feat]: Netdata native collector for AWS cloudwatch metrics

### ISSUE THREAD

**Issue by shyamvalsan** [netdata/netdata#17728]
### Problem

To collect and visualize AWS cloudwatch metrics, the currently supported way is to utilize the prometheus exporter for cloudwatch. 

It would be ideal if there was a native collector in Netdata that can do this in a more optimal and usable way.

### Description

Users have suggested [YACE](https://github.com/nerdswords/yet-another-cloudwatch-exporter) as the model for how this collector should work. It supports almost all relevant AWS services that users may wish to monitor.

### Importance

must have

### Value proposition

1. A large % of Netdata users host their infra on AWS and this would be significantly beneficial. Especially if there are pre-configured alerts, summary dashboards etc. for the AWS metrics.

**Comment by brodonalds** [comment]
Take note that metrics get really expensive with API calls. If Netdata had a way to take some of the sting out like how logicmonitor or others do in the configuration side, that would make it easier to figure out which metrics end users may want or not.

**Comment by shyamvalsan** [comment]
> Take note that metrics get really expensive with API calls. If Netdata had a way to take some of the sting out like how logicmonitor or others do in the configuration side, that would make it easier to figure out which metrics end users may want or not.

Fair point. This should definitely be built in. I think Netdata needs to be "opinionated" about how AWS monitoring needs to be done. In terms of granularity, coverage etc. 

I was thinking maybe there are different configuration profiles that the users can choose from (and of course allow the user to customize if they know how to): 
- level1 (basic visibility + low cost)
- level2 (medium visibility + medium cost)
- level3 (extensive visibility + high cost)
